catroid/src/org/catrobat/catroid/content/Look.java

```diff
@@ -370,6 +379,24 @@ public void changeBrightnessInUserInterfaceDimensionUnit(float changePercent) {
 		setBrightnessInUserInterfaceDimensionUnit(getBrightnessInUserInterfaceDimensionUnit() + changePercent);
 	}
 
+	public float getColorInUserInterfaceDimensionUnit() {
+		return hue * 200.0f;
```

Comment:
200 shows at several places in two methods:

- `getColorInUserInterfaceDimensionUnit`
- `setColorInUserInterfaceDimensionUnit`

Could you create a constant for that please?